packages/rspack/tests/Stats.test.ts

```diff
@@ -163,7 +190,7 @@ describe("Stats", () => {
 		    entry ./fixtures/a
 		./fixtures/a.js [876] {main}
 		  entry ./fixtures/a
-		rspack compiled successfully (f0a86c7e70b0de037daf)"
+		rspack compiled successfully (1788bb8a658f2310fbf6)"
```

Comment:
hash change is caused by https://github.com/web-infra-dev/rspack/pull/3893/files#diff-a3ed34d20f0b15a073bdeaac181d27e0fe74d19b63f7a0eb6cc1868192ef6dd9L66-R70, the name will be hashed in js_chunk_hash